collect_env.py

```diff
@@ -519,6 +522,16 @@ def is_xnnpack_available():
     else:
         return "N/A"
 
+def get_env_vars():
+    env_vars = ''
+    BLACK_LIST=('S3_SECRET_ACCESS_KEY', 'VLLM_API_KEY', 'S3_ACCESS_KEY_ID')
+    for k, v in os.environ.items():
+        if k in BLACK_LIST:
```

Comment:
```suggestion
    secret_terms=('secret', 'token', 'api', 'access', 'password')
    for k, v in os.environ.items():
        if any(term in k.lower() for term in secret_terms):
```

this might make it a bit more robust to other env vars that are added later

Reply:
Good point. Done